### ISSUE TITLE
Perfomance 

### ISSUE THREAD

**Issue by DaryaTonkykh** [povoroznykmarta/web-design-2-independent-project#8]
Your performance testing was a 96 so it is pretty good over all. Sorry should have added to your accessibility but according to the test it could be approved upon, maybe add some aria labels.
